frontend/public/module/k8s/index.ts

```diff
@@ -617,3 +617,24 @@ export type GroupVersionKind = string;
  * Maintains backwards-compatibility with references using the `kind` string field.
  */
 export type K8sResourceKindReference = GroupVersionKind | string;
+
+export type EventInvolvedObject = {
```

Comment:
Nit: isn't it better to add `public/module/k8s/event.ts` and add `export * from './event';` at the top?

Reply:
I guess you're right. Moved that type to `event.ts`